community.html

```diff
@@ -43,7 +43,7 @@ <h2>Help others</h2>
   <h2>Help the project</h2>
   <ul>
     <li class="package"><h3>Package an App</h3>
-      <p>Check out our <a href="https://docs.sandstorm.io/en/latest/vagrant-spk/packaging-tutorial/">tutorial</a> to get started packaging apps for the <a href="https://apps.sandstorm.io/">App Market</a>. If you get stuck, <a href="https://groups.google.com/group/sandstorm-dev"> email us</a> for help or hop on <a href="https://kiwiirc.com/client/irc.libera.chat/?channel=#sandstorm">IRC</a>.</p>
+      <p>Check out our <a href="https://docs.sandstorm.io/en/latest/vagrant-spk/packaging-tutorial/">tutorial</a> to get started packaging apps for the <a href="https://apps.sandstorm.io/">App Market</a>. If you get stuck, <a href="https://groups.google.com/group/sandstorm-dev"> email us</a> for help or hop on <a href="https://matrix.to/#/#sandstorm:libera.chat">Matrix chat</a>.</p>

```

Comment:
I can't tag a line you didn't change in the review, but IRC is mentioned on line 54, and should be updated. I would also say it takes place "on Matrix" and not "on Matrix chat" because that's redundant.